presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPickTableLayout.java

```diff
@@ -40,40 +42,58 @@
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.constrainedTableScanWithTableLayout;
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.filter;
 import static com.facebook.presto.sql.planner.iterative.rule.test.PlanBuilder.expression;
+import static io.airlift.testing.Closeables.closeAllRuntimeException;
 
 public class TestPickTableLayout
-        extends BaseRuleTest
 {
     private static final TableLayoutHandle DUMMY_TABLE_LAYOUT_HANDLE = new TableLayoutHandle(new ConnectorId("tpch"),
             TestingTransactionHandle.create(),
             new TpchTableLayoutHandle(new TpchTableHandle("local", "nation", 1.0), Optional.empty()));
 
     private RuleSet pickTableLayout;
     private TableHandle nationTableHandle;
+    private TableHandle ordersTableHandle;
+    private RuleTester tester;
+
+    @BeforeClass
+    public final void setUp()
+    {
+        tester = new RuleTester();
+    }
+
+    @AfterClass(alwaysRun = true)
+    public final void tearDown()
+    {
+        closeAllRuntimeException(tester);
+        tester = null;
+    }
 
     @BeforeMethod
     public void setUpPerMethod()
     {
-        pickTableLayout = new PickTableLayout(tester().getMetadata(), new SqlParser());
+        pickTableLayout = new PickTableLayout(tester.getMetadata(), new SqlParser());
 
-        ConnectorId connectorId = tester().getCurrentConnectorId();
+        ConnectorId connectorId = tester.getCurrentConnectorId();
         nationTableHandle = new TableHandle(
                 connectorId,
                 new TpchTableHandle(connectorId.toString(), "nation", 1.0));
+        ordersTableHandle = new TableHandle(
+                connectorId,
+                new TpchTableHandle(connectorId.toString(), "orders", 1.0));
```

Comment:
This change is unrelated to "Use RuleTester instance explicitly in PickTableLayout tests" commit.